aws/aws.go

```diff
@@ -424,7 +424,21 @@ func GetAllResources(targetRegions []string, excludeAfter time.Time, resourceTyp
 			}
 		}
 		// End Lambda Functions
+		// Dynamo DB tables
+		DynamoDB := DynamoDB{}
+		if IsNukeable(DynamoDB.ResourceName(), resourceTypes) {
+			tablenames, err := getAllDynamoTables(session)
```

Comment:
With `cloud-nuke`, we need to respect the `older-than` parameter... It looks like this nukes all DynamoDB tables, rather than those older than the date requested by the user?